src/consts/consts.py

```diff
@@ -144,7 +144,7 @@ def all(cls):
 )
 
 # DISK SIZES
-DISK_SIZE_120GB = 120 * 2**30
+DISK_SIZE_120GB = 120 * 10**9
```

Comment:
Maybe declare a 1GB unit const and use it across the project?

Reply:
sure, done